### ISSUE TITLE
pip-review does not find updates, pip list --outdated does

### ISSUE THREAD

**Issue by caedn** [jgonggrijp/pip-review#18]
```
$ pip2 list --outdated                                                                                                                                                           
arrow (0.8.0) - Latest: 0.10.0 [sdist]
beets (1.3.19) - Latest: 1.4.1 [sdist]
cheat (2.1.26) - Latest: 2.1.27 [sdist]
Cython (0.25.1) - Latest: 0.25.2 [wheel]
[...]

$ pip-review
Everything up-to-date

$ python2 -m pip_review
Everything up-to-date

$ pip2 show pip_review                                                                                                                                                            
Name: pip-review
Version: 0.5
[...]
Location: /usr/local/lib/python2.7/site-packages
Requires: packaging

$ pip2 --version
pip 9.0.1 from /usr/local/lib/python2.7/site-packages (python 2.7)

$ uname -rs
Darwin 16.1.0
```

This happens with Python 2.7.12, the builtin version of macOS and with Python 3.6.0rc1 as well.

As a workaround I've used: `pip2 list --outdated|cut -d' ' -f1|xargs pip2 install --upgrade`.
If you want it to be interactive you could do something like `pip2 list --outdated|vipe|cut -d' ' -f1|xargs pip2 install --upgrade`.

**Comment by jgonggrijp** [comment]
Just for completeness: does this only happen with global packages, or also in a virtual environment? In the former case, using a virtual environment may be another possible workaround.

**Comment by k4rtik** [comment]
Facing the same issue.

```
$ pip2 list -o
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
autopep8 (1.2.4) - Latest: 1.3.1 [sdist]
... [many more packages here]
wrapt (1.10.8) - Latest: 1.10.10 [sdist]
```

while

```
$ pip2 install pip-review
Collecting pip-review
  Using cached pip_review-0.5.2-py2.py3-none-any.whl
Requirement already satisfied: packaging in /usr/local/lib/python2.7/site-packages (from pip-review)
Requirement already satisfied: pip in /usr/local/lib/python2.7/site-packages (from pip-review)
Requirement already satisfied: six in /usr/local/lib/python2.7/site-packages (from packaging->pip-review)
Requirement already satisfied: pyparsing in /usr/local/lib/python2.7/site-packages (from packaging->pip-review)
Installing collected packages: pip-review
Successfully installed pip-review-0.5.2
$ pip-review
Everything up-to-date
```

About @jgonggrijp's question, I am not using virtual environments.

**Comment by jgonggrijp** [comment]
Fixed in [version 1.0](https://github.com/jgonggrijp/pip-review/releases/tag/1.0).
